iosApp/iosApp/Localizable.xcstrings

```diff
@@ -595,96 +595,96 @@
         }
       }
     },
-    "%@ arriving in %@ min" : {
+    "%@ arriving in %d min" : {
```

Comment:
using the strings as-is on android gave a warning about passing an int when the formatter expected a string.